{"uid":"92459148fd3d630b","name":"Test valid login with fallback OCR and AI suggestions","fullName":"tests.test_login#test_login_success","historyId":"86600b41e86dab7ca99f472fa61e4b05","time":{"start":1751975547772,"stop":1751975604020,"duration":56248},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":39,"retriesStatusChange":true,"beforeStages":[{"name":"driver","time":{"start":1751975538438,"stop":1751975547772,"duration":9334},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false,"attachmentStep":false,"stepsCount":0}],"testStage":{"status":"passed","steps":[],"attachments":[{"uid":"34411ca2c8886bff","name":"Test Result","source":"34411ca2c8886bff.txt","type":"text/plain","size":17},{"uid":"6073742231f95512","name":"stdout","source":"6073742231f95512.txt","type":"text/plain","size":531}],"parameters":[],"attachmentsCount":2,"shouldDisplayMessage":false,"hasContent":true,"attachmentStep":false,"stepsCount":0},"afterStages":[{"name":"driver::0","time":{"start":1751975604020,"stop":1751975604292,"duration":272},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false,"attachmentStep":false,"stepsCount":0}],"labels":[{"name":"epic","value":"Login Flow"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_login"},{"name":"host","value":"DELL"},{"name":"thread","value":"24452-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_login"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"ea3eef2dbc5f1c","status":"passed","time":{"start":1751974976621,"stop":1751975032522,"duration":55901}},{"uid":"a7a994a464d6155d","status":"failed","statusDetails":"AssertionError: ❌ Test failed: 'NoneType' object has no attribute 'click'","time":{"start":1751974746451,"stop":1751974775254,"duration":28803}},{"uid":"a49fa69f09df2e","status":"failed","statusDetails":"AssertionError: ❌ Test failed: 'dashboard_title'","time":{"start":1751974209326,"stop":1751974272381,"duration":63055}},{"uid":"e9caee4b004caa00","status":"failed","statusDetails":"AssertionError: ❌ Test failed: 'dashboard_title'","time":{"start":1751973998022,"stop":1751974055644,"duration":57622}},{"uid":"d5ceb6b6fbbb257d","status":"failed","statusDetails":"AssertionError: ❌ Test failed: 'dashboard_title'","time":{"start":1751973384093,"stop":1751973432912,"duration":48819}},{"uid":"843e9b77498262ef","status":"failed","statusDetails":"AssertionError: ❌ Test failed: 'dashboard_title'","time":{"start":1751973094893,"stop":1751973137840,"duration":42947}},{"uid":"cd3a7aeba870bead","status":"failed","statusDetails":"AssertionError: ❌ Test failed: ❌ Next button not found\nassert None is not None","time":{"start":1751972769187,"stop":1751972801189,"duration":32002}},{"uid":"2334ce803d4ec80","status":"failed","statusDetails":"AssertionError: ❌ Test failed: ❌ Next button not found\nassert None is not None","time":{"start":1751972632433,"stop":1751972664370,"duration":31937}},{"uid":"10572603aa32973b","status":"failed","statusDetails":"AssertionError: ❌ Test failed: ❌ Next button not found\nassert None is not None","time":{"start":1751972502040,"stop":1751972535591,"duration":33551}},{"uid":"1e7e904dc6e8acfe","status":"failed","statusDetails":"AssertionError: ❌ Test failed: ❌ Phone number input not found\nassert None is not None","time":{"start":1751972235386,"stop":1751972249683,"duration":14297}},{"uid":"b5376e8a11ae389b","status":"failed","statusDetails":"AssertionError: ❌ Test failed: 'WebDriver' object has no attribute 'find_element_by_xpath'","time":{"start":1751971984219,"stop":1751971986973,"duration":2754}},{"uid":"777535036be283c8","status":"failed","statusDetails":"AssertionError: ❌ Test failed: 'WebDriver' object has no attribute 'find_element_by_xpath'","time":{"start":1751971939266,"stop":1751971941155,"duration":1889}},{"uid":"9ed649d1c4378c5d","status":"failed","statusDetails":"AssertionError: ❌ Test failed: ❌ Phone number input not found\nassert None is not None","time":{"start":1751971771336,"stop":1751971785571,"duration":14235}},{"uid":"6045b4144f0f2955","status":"failed","statusDetails":"AssertionError: ❌ Test failed: ❌ Next button not found\nassert None is not None","time":{"start":1751971515210,"stop":1751971549652,"duration":34442}},{"uid":"654c5b6d3e59306a","status":"failed","statusDetails":"AssertionError: ❌ Test failed: ❌ Phone number input not found\nassert None is not None","time":{"start":1751971102900,"stop":1751971117507,"duration":14607}},{"uid":"50844cc88870c272","status":"failed","statusDetails":"AssertionError: ❌ Test failed: ❌ Phone number input not found\nassert None is not None","time":{"start":1751970974774,"stop":1751970991243,"duration":16469}},{"uid":"d994ac1f45ce425d","status":"failed","statusDetails":"AssertionError: ❌ Login test failed: phone_number_input input not found\nassert None is not None","time":{"start":1751970027802,"stop":1751970051676,"duration":23874}},{"uid":"5da9920b4b1b19e5","status":"failed","statusDetails":"AssertionError: ❌ Login test failed: phone_number_input input not found\nassert None is not None","time":{"start":1751969976029,"stop":1751969999353,"duration":23324}},{"uid":"97ee11452473c9f0","status":"failed","statusDetails":"AssertionError: ❌ Login test failed: Username input not found\nassert None is not None","time":{"start":1751969498331,"stop":1751969521950,"duration":23619}},{"uid":"a32c3a298b9f06ca","status":"failed","statusDetails":"AssertionError: ❌ Login test failed: Username input not found\nassert None is not None","time":{"start":1751969353399,"stop":1751969376814,"duration":23415}},{"uid":"c915bba15953fb24","status":"broken","statusDetails":"NameError: name 'APK_PATH' is not defined","time":{"start":1751969322216,"stop":1751969322216,"duration":0}},{"uid":"f8a386aaba3f1c7a","status":"failed","statusDetails":"AssertionError: ❌ Login test failed: Username input not found\nassert None is not None","time":{"start":1751969191655,"stop":1751969214692,"duration":23037}},{"uid":"f02aaf13399a3adb","status":"failed","statusDetails":"Failed: Failed to initialize Appium driver: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.krishivaas' application. Consider checking the driver's troubleshooting documentation. Original error: Cannot start the 'com.krishivaas' application. Consider checking the driver's troubleshooting documentation. Original error: Activity name '.com.krishivaas.app.MainActivity' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.krishivaas' application. Consider checking the driver's troubleshooting documentation. Original error: Cannot start the 'com.krishivaas' application. Consider checking the driver's troubleshooting documentation. Original error: Activity name '.com.krishivaas.app.MainActivity' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\n    at getResponseForW3CError (C:\\Users\\Pramo\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\@appium\\base-driver\\lib\\protocol\\errors.js:1143:9)\n    at asyncHandler (C:\\Users\\Pramo\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\@appium\\base-driver\\lib\\protocol\\protocol.js:487:57)","time":{"start":1751969078432,"stop":1751969078432,"duration":0}},{"uid":"4279b2d2e33f03b3","status":"failed","statusDetails":"Failed: Failed to initialize Appium driver: Message: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.krishivaas' application. Consider checking the driver's troubleshooting documentation. Original error: Cannot start the 'com.krishivaas' application. Consider checking the driver's troubleshooting documentation. Original error: Activity name '.com.krishivaas.app.MainActivity' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Cannot start the 'com.krishivaas' application. Consider checking the driver's troubleshooting documentation. Original error: Cannot start the 'com.krishivaas' application. Consider checking the driver's troubleshooting documentation. Original error: Activity name '.com.krishivaas.app.MainActivity' used to start the app doesn't exist or cannot be launched! Make sure it exists and is a launchable activity\n    at getResponseForW3CError (C:\\Users\\Pramo\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\@appium\\base-driver\\lib\\protocol\\errors.js:1143:9)\n    at asyncHandler (C:\\Users\\Pramo\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\@appium\\base-driver\\lib\\protocol\\protocol.js:487:57)","time":{"start":1751968970841,"stop":1751968970841,"duration":0}},{"uid":"e8d372d791a5d512","status":"failed","statusDetails":"Failed: Failed to initialize Appium driver: Message: An unknown server-side error occurred while processing the command. Original error: The application at 'C:\\Users\\Pramo\\Krishivaas_Farmer_UI_RN(V0.12.34).apk' does not exist or is not accessible\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: The application at 'C:\\Users\\Pramo\\Krishivaas_Farmer_UI_RN(V0.12.34).apk' does not exist or is not accessible\n    at getResponseForW3CError (C:\\Users\\Pramo\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\@appium\\base-driver\\lib\\protocol\\errors.js:1143:9)\n    at asyncHandler (C:\\Users\\Pramo\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\@appium\\base-driver\\lib\\protocol\\protocol.js:487:57)","time":{"start":1751968884275,"stop":1751968884275,"duration":0}},{"uid":"8de261174e60944e","status":"failed","statusDetails":"Failed: Failed to initialize Appium driver: Message: An unknown server-side error occurred while processing the command. Original error: The application at 'C:\\Users\\Pramo\\Krishivaas_Farmer_UI_RN(V0.12.34).apk' does not exist or is not accessible\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: The application at 'C:\\Users\\Pramo\\Krishivaas_Farmer_UI_RN(V0.12.34).apk' does not exist or is not accessible\n    at getResponseForW3CError (C:\\Users\\Pramo\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\@appium\\base-driver\\lib\\protocol\\errors.js:1143:9)\n    at asyncHandler (C:\\Users\\Pramo\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\@appium\\base-driver\\lib\\protocol\\protocol.js:487:57)","time":{"start":1751968856887,"stop":1751968856887,"duration":0}},{"uid":"afb56745bbe79cb6","status":"failed","statusDetails":"Failed: Failed to initialize Appium driver: Message: An unknown server-side error occurred while processing the command. Original error: Could not find a connected Android device in 20000ms\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Could not find a connected Android device in 20000ms\n    at getResponseForW3CError (C:\\Users\\Pramo\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\@appium\\base-driver\\lib\\protocol\\errors.js:1143:9)\n    at asyncHandler (C:\\Users\\Pramo\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\@appium\\base-driver\\lib\\protocol\\protocol.js:487:57)","time":{"start":1751968673208,"stop":1751968673208,"duration":0}},{"uid":"342c20a7b87ac402","status":"failed","statusDetails":"Failed: Failed to initialize Appium driver: Message: An unknown server-side error occurred while processing the command. Original error: Could not find a connected Android device in 20000ms\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Could not find a connected Android device in 20000ms\n    at getResponseForW3CError (C:\\Users\\Pramo\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\@appium\\base-driver\\lib\\protocol\\errors.js:1143:9)\n    at asyncHandler (C:\\Users\\Pramo\\AppData\\Roaming\\npm\\node_modules\\appium\\node_modules\\@appium\\base-driver\\lib\\protocol\\protocol.js:487:57)","time":{"start":1751968375855,"stop":1751968375855,"duration":0}},{"uid":"5c71c1995ed2982f","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource","time":{"start":1751968154544,"stop":1751968154544,"duration":0}},{"uid":"5d73406b26a6217","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource","time":{"start":1751967978159,"stop":1751967978159,"duration":0}},{"uid":"44e53cfe8aad5025","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource","time":{"start":1751967911556,"stop":1751967911556,"duration":0}},{"uid":"4aa20625cbaf3936","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: The requested resource could not be found, or a request was received using an HTTP method that is not supported by the mapped resource","time":{"start":1751967768832,"stop":1751967768832,"duration":0}},{"uid":"ab530de726167587","status":"broken","statusDetails":"TypeError: missing 1 required keyword-only argument: 'options' (instance of driver `options.Options` class)","time":{"start":1751967520075,"stop":1751967520075,"duration":0}},{"uid":"bdcd3914c64ddbf8","status":"broken","statusDetails":"TypeError: WebDriver.__init__() got an unexpected keyword argument 'desired_capabilities'","time":{"start":1751967114311,"stop":1751967114311,"duration":0}},{"uid":"894f2221e8796039","status":"broken","statusDetails":"TypeError: WebDriver.__init__() got an unexpected keyword argument 'comman_executor'","time":{"start":1751967096153,"stop":1751967096153,"duration":0}},{"uid":"c50f00bd7139a907","status":"broken","statusDetails":"TypeError: WebDriver.__init__() got an unexpected keyword argument 'desired_capabilities'","time":{"start":1751967045923,"stop":1751967045923,"duration":0}},{"uid":"25c7f24bc168c37a","status":"broken","statusDetails":"TypeError: missing 1 required keyword-only argument: 'options' (instance of driver `options.Options` class)","time":{"start":1751966971757,"stop":1751966971757,"duration":0}},{"uid":"d82165167bd54d15","status":"broken","statusDetails":"TypeError: missing 1 required keyword-only argument: 'options' (instance of driver `options.Options` class)","time":{"start":1751966821443,"stop":1751966821443,"duration":0}},{"uid":"50712d3dd60102ce","status":"broken","statusDetails":"TypeError: missing 1 required keyword-only argument: 'options' (instance of driver `options.Options` class)","time":{"start":1751966324348,"stop":1751966324348,"duration":0}}],"categories":[],"tags":[]},"source":"92459148fd3d630b.json","parameterValues":[]}